.github/workflows/build-cirros.yaml

```diff
@@ -21,6 +21,8 @@ jobs:
       ARCHES: ${{ matrix.arch }}
       BOOTTEST: "true"
       QUIET: 1
+      CI_BUILD: "true"
+      CI_BUILD_OUT: "$PWD/build-ci/"
```

Comment:
I think that this doesn't work because $PWD isn't expanded when you use this later. Maybe just go with a relative path? If an absolute path is needed, maybe the best solution is still to just specify the suffix here and add in the $PWD in the build script.